README.rst

```diff
@@ -17,46 +17,49 @@ Pytest container
            :alt: PyPI
            :target: https://pypi.org/project/pytest-container/
 
-A simple `pytest <https://pytest.org>`_ plugin to test Container Images
-via pytest fixtures and `testinfra <https://testinfra.readthedocs.io/en/latest/>`_.
-
-This module provides a set of fixtures and helper functions to ease testing of
-container images leveraging `testinfra
-<https://testinfra.readthedocs.io/en/latest/>`_. Assuming you want to
-automatically spin up a container for a test, then the `container` fixture will
-do exactly that (plus it will cleanup after itself):
+``pytest_container`` is a small and simple `pytest <https://pytest.org>`_ plugin
+to test container images via pytest fixtures and `testinfra
+<https://testinfra.readthedocs.io/en/latest/>`_. It takes care of all the boring
+tasks, like spinning up containers, finding free ports and cleaning up after
+tests, and allows you to focus on implementing the actual tests.
+
+Some of the features include:
```

Comment:
These are not features but tasks the plugin performs, no? In this case, I'd suggest something like this:

```suggestion
The plugin automates the following tasks:
```